docs/source/getting_started/projects.rst

```diff
@@ -0,0 +1,127 @@
+========
+Projects
+========
+
+A Project is the basic unit of collaboration and access on the Citrine platform.
+They are containers for other resources, such as Datasets and modules.
```

Comment:
```suggestion
They are collections of users, who share ownership and access rights to other resources such as Datasets and modules.
```

Reply:
I'm not sure that this is useful as the primary framing, since the same set of users can conceivably create multiple projects.